core/spec/validators/db_maximum_length_validator_spec.rb

```diff
@@ -1,22 +1,33 @@
 require 'spec_helper'
 
 Spree::Product.class_eval do
```

Comment:
@priyank-gupta @mauazua Couldn't we just remove this spec? It's not really testing anything properly, as it is operating on overridden class.

In addition, it is not restoring Product class, so every further spec also uses overridden version.